mssqlcli/mssqlcliclient.py

```diff
@@ -24,7 +24,13 @@ def __init__(self, sql_tools_client, server_name, user_name, password,
                  authentication_type=u'SqlLogin', database=u'master', owner_uri=None, multiple_active_result_sets=True,
                  encrypt=None, trust_server_certificate=None, connection_timeout=None, application_intent=None,
                  multi_subnet_failover=None, packet_size=None, **kwargs):
+
         self.server_name = server_name
+        if ',' in server_name:
+            self.host, self.port = self.server_name.split(',')
+        else:
+            self.host = server_name
+            self.port = 1433
```

Comment:
For practical purposes, 1433 is correct. However, SQL Server supports other transports, named pipes and shared memory, which use different ports. Also, the dedicated admin connection uses 1434. However, we can ignore this and fix if we ever get any complaints. I doubt we will :)